src/lib/utils.test.js

```diff
@@ -12,147 +12,226 @@ import {
 	checkIfCookieIsOld,
 	getTimestamp,
 	getConsentsCount
-} from './utils';
-import config from './config';
+} from "./utils";
+import config from "./config";
 
-let vendorList = {vendors: [{id: 1}, {id: 3}, {id: 4}]};
+let vendorList = { vendors: [{ id: 1 }, { id: 3 }, { id: 4 }] };
 
-describe('utils', () => {
-	describe('checkReprompt', () => {
+describe("utils", () => {
+	describe("checkReprompt", () => {
 		let { repromptOptions } = config;
 
-		it('returns true if there are no cookies set', () => {
+		it("returns true if there are no cookies set", () => {
 			const vendorConsent = {
-				"purposeConsents": {
+				purposeConsents: {
 					"1": false
 				},
-				"vendorConsents": {
-					"1": false,
+				vendorConsents: {
+					"1": false
 				}
 			};
 			const publisherConsent = {
-				"customPurposes": {
-					"1": false,
+				customPurposes: {
+					"1": false
 				},
-				"standardPurposes": {
-					"1": false,
+				standardPurposes: {
+					"1": false
 				}
 			};
-			expect(checkReprompt(repromptOptions, vendorList, vendorConsent, publisherConsent)).eq(true);
+			expect(
+				checkReprompt(
+					repromptOptions,
+					vendorList,
+					vendorConsent,
+					publisherConsent
+				)
+			).eq(true);
 		});
 
-		it('returns true if atleast one cookie is not set', () => {
+		it("returns true if atleast one cookie is not set", () => {
```

Comment:
@LassaLek  `returns true if atleast one cookie is not set` fails and returns `false`